lib/core/src/animation/animator/continuous.rs

```diff
@@ -114,8 +114,8 @@ impl ContinuousAnimator {
         let event_loop      = EventLoop::new();
         let data            = ContinuousAnimatorData::new(f);
         let weak_data       = Rc::downgrade(&data);
-        let callback_handle = event_loop.add_callback(move |absolute_time_ms| {
-            weak_data.upgrade().map(|data| data.on_animation_frame(absolute_time_ms));
+        let callback_handle = event_loop.add_callback(move |absolute_time_ms:&f64| {
```

Comment:
This is probably a compiler error, because it should be capable of inferring the type.

There is a discussion about it here: https://github.com/rust-lang/rust/issues/58639